### ISSUE TITLE
update static files example to deny directory listings

### ISSUE THREAD

**Issue by ktdreyer** [twisted/klein#277]
I was writing my first app with Klein, and I wanted to serve some CSS and Javascript files. I read this example: https://klein.readthedocs.io/en/latest/examples/staticfiles.html

It shows me how to open up a directory, but anyone can browse the whole directory on disk to discover all the contents. I don't want to enable directory listings, and I just want to serve specific static files.

The [twisted-web mailing list](https://twistedmatrix.com/pipermail/twisted-web/2006-August/002924.html) has this suggestion:

```python
class FileNoDir(static.File):
    def directoryListing(self):
        return None
```

It would be great to do this in the Klein example documentation so developers do not deploy apps that accidentally expose more information than intended.

**Comment by Addvilz** [comment]
You should probably raise `NotFound` instead of returning None - using None as response will result in `processing failed` and HTTP 500. Using NotFound will, as expected, result in 404.

```
from werkzeug.exceptions import NotFound

# [...]

class FileNoDir(File):
    def directoryListing(self):
        raise NotFound()
```

**Comment by ktdreyer** [comment]
Thanks for the suggestion. I included your suggestion in PR #387 .
